efficientdet/backbone/efficientnet_builder.py

```diff
@@ -325,3 +325,54 @@ def build_model_base(images, model_name, training, override_params=None):
   model = efficientnet_model.Model(blocks_args, global_params, model_name)
   features = model(images, training=training, features_only=True)
   return features, model.endpoints
+
+
+def get_model(model_name,
```

Comment:
How about move this get_model to "https://github.com/google/automl/blob/master/efficientdet/backbone/backbone_factory.py"? Then you don't need to define this in each builder file?